ui/src/setupProxy.js

```diff
@@ -1,18 +0,0 @@
-const { createProxyMiddleware } = require('http-proxy-middleware');
-
-module.exports = (app) => {
-  app.use(
-    '/mxcube/api',
-    createProxyMiddleware({
-      target: 'http://127.0.0.1:8081',
-      ws: false,
-    }),
-  );
-  app.use(
-    '/socket.io/*',
-    createProxyMiddleware({
-      target: 'http://127.0.0.1:8081',
-      ws: false,
```

Comment:
Not quite sure why this was configured like this, since I used to get socket.io errors in development. I switched to `ws: true` and `target: 'ws://127.0.0.1:8081` and the error is now gone.